src/main/java/de/zalando/aruha/nakadi/controller/EventTypeController.java

```diff
@@ -67,12 +73,13 @@ public EventTypeController(final EventTypeRepository eventTypeRepository, final
 
         try {
             validateSchema(eventType);
+            validatePartitioningStrategy(eventType);
             eventTypeRepository.saveEventType(eventType);
             topicRepository.createTopic(eventType.getName());
             return status(HttpStatus.CREATED).build();
-        } catch (final InvalidEventTypeException e) {
-            return create(e.asProblem(), nativeWebRequest);
-        } catch (final DuplicatedEventTypeNameException e) {
+        } catch (final InvalidEventTypeException | NoSuchPartitioningStrategyException |
+                DuplicatedEventTypeNameException e) {
+            LOG.warn(e.getMessage(), e);
```

Comment:
I think we never discussed that in our team.
But I see your point. It should not really be a warning as it is just usual work of an application. So Debug or Info will fit better here. Will fix that.